libs/ledger/include/ledger/identifier.hpp

```diff
@@ -37,7 +37,6 @@ class Identifier
 public:
   enum class Type
   {
-    INVALID,
```

Comment:
Feel like we should keep this type because this can be default initialised

Reply:
Done